<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>جذر</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #dad3c3;</string>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Egypt"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #c2baa4;
border-radius: 5;
font-size: 18px;
</string>
    </property>
    <column>
     <property name="text">
      <string>Word</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stem</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #a69f8b;
border-radius: 5;
font-size: 20px;
</string>
    </property>
    <property name="text">
     <string>Get Stem</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #c2baa4;
border-radius: 5;
padding: 5 5 5 5 ;
font-size: 18px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
